data.sql

```diff
@@ -1,5 +1,12 @@
 /* Populate database with sample data. */
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (1, 'Luna', '2020-02-03', 0, TRUE, 10.23);
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (2, 'Daisy', '2018-11-15', 2, TRUE, 8);
```

Comment:
- [ ] Please note that you will be querying this table for the rest of the week and if you have a different name from the one given in the project requirements, it might cause some confusion for the reviewers when the queries and database structure become complex. Kindly change the name of this animals to the ones given in the project requirements.